### ISSUE TITLE
Option Function Demo

### ISSUE THREAD

**Issue by smtir** [evertiro/historical-redux2#114]
Can u create all field function demo? That's better for new user.

**Comment by evertiro** [comment]
What do  you mean? All fields are demoed in the framework itself.

**Comment by smtir** [comment]
I have seen ever great option but problem is I don't know how to retrieve multiple text option, Sortable Text Option, Select Hide Below Option, Multi Checkbox Option. So if u create example those function how to retrieve then its better for me and others new users.
Thanks

**Comment by evertiro** [comment]
Ahh. That makes more sense. We actually used to include a sample template, but it was removed when the majority of the community indicated they'd rather have the option to use the framework as a git submodule. However... it's quite simple really. In your functions.php (or some other file that's called before any options are used), add the following where _$myvariable_ is the variable you will use to reference the options, and _myoptions_ is the value you assigned to _$args['opt_name']_ in your config file:

``` php
$myvariable = get_option('myoptions');
```

Then, to reference a specific variable (we'll use _mycolor_ for our example), you would simply access it as follows, making sure that a field with a corresponding ID exists in your config:

``` php
echo $myvariable['mycolor'];
```
